compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1229,11 +1232,23 @@ static void lhsPeephole(TR::CodeGenerator *cg, TR::Instruction *storeInstruction
    // Found the pattern:
    //   stw/std rX, ...
    //   lwz/ld rY, ...
-   // and will remove lwz/ld, which will result in:
-   //   stw/std rX, ...
+   // and will remove lwz, which will result in:
+   //   stw rX, ...
+   //   rlwinm rY, rX, 0, 0xffffffff
+   // and then the rlwinm peephole should run on the resulting rlwinm
```

Comment:
Get rid of this comment line, unlike the case for `mr`, there is no `rlwinm` peephole.